src/middlewares/auth.ts

```diff
@@ -34,32 +26,13 @@ export function authMiddleware(req: RequestExtended, res: Response, next: NextFu
 
   // check for refresh token in body?
   if ('refresh_token' in req.query) {
-    refresh_token = {
-      value: req.query.refresh_token as string,
-      type: 'query'
-    }
-    req.refresh_token = refresh_token
+    req.refresh_token = req.query.refresh_token as string
+    delete req.query.refresh_token
   }
 
-  // -------------------------------------
-  // COOKIES
-  // -------------------------------------
-  const cookiesInUse = COOKIES.SECRET ? req.signedCookies : req.cookies
-
-  if ('refresh_token' in cookiesInUse) {
-    refresh_token = {
-      value: cookiesInUse.refresh_token,
-      type: 'cookie'
-    }
-    req.refresh_token = refresh_token
-  }
-
-  if ('permission_variables' in cookiesInUse) {
-    try {
-      req.permission_variables = getPermissionVariablesFromCookie(req)
-    } catch (err) {
-      return res.boom.unauthorized(err.message)
-    }
+  if ('permission_variables' in req.query) {
+    req.permission_variables = JSON.parse(req.query.permission_variables as string)
+    delete req.query.permission_variables
```

Comment:
Why do we do this? Looks like a security issue if the permission variables can be set using a query variable.